client/a8c-for-agencies/sections/partner-directory/dashboard/index.tsx

```diff
@@ -261,7 +261,7 @@ const PartnerDirectoryDashboard = () => {
 						{
 							// Application approved, but the Directory page is not available yet
 							application.key === 'approved' && ! brandMeta.isAvailable
-								? translate( 'This partner directory is launching soon.' )
+								? translate( 'This Partner Directory is launching soon.' )
```

Comment:
Related to the question before, either we titlecase "Partner Directory" _everywhere_, or we sentence case it _everywhere_. In this PR there's currently a mix.

Reply:
This check uses the `brandMeta.isAvailable` property, which is now set to `true` for all four partner directories. It won't display now, but it seems this code makes sense as it would be used as soon as we've added another directory with `isAvailable` set to `false`.